spark/dpl-interpreter/src/main/java/com/teragrep/pth_07/ui/elements/table_dynamic/DTTableDatasetNg.java

```diff
@@ -99,12 +101,19 @@ public void setParagraphDataset(Dataset<Row> rowDataset) {
 
         try {
             lock.lock();
+            // Reset draw when schema changes
+            if(!schemaHeaders.schema().equals(rowDataset.schema())){
```

Comment:
can schema change between runs as internalInterpret is rerun foreach of the interpretations and schema can not change once spark has determined it